ref/undescribedfaq.md

```diff
@@ -0,0 +1,54 @@
+
+A note to preface: Raising an inducer is not hard, but each individual attempt has low odds of success. The biggest reason experts are more likely to succeed is not any particular technique but because they search harder and collect galls in larger numbers than any individual amateur. As a community, we can distribute that effort across many observers. So while you might not be successful yourself, you are still contributing to the process that builds the knowledge we need for someone to eventually succeed. 
+
+So if you find a gall you determine to be undescribed (or perhaps a described gall that is of interest for other reasons), what should you do?
+The answer varies significantly depending on the taxon of the inducer. 
+
+## 1. Broadly place the taxon of the inducer. 
+  For a truly new, unknown gall, or a gall listed as Unknown on Gallformers, the first step is to figure out what the likely taxon of the inducer is. This can sometimes be done with obvious external features, like rust fruiting bodies or mite erineum, but in general it requires dissection. Carefully cut apart the gall to avoid destroying any larvae that might exist inside, and photograph both the structure of the gall and the larvae. This information should allow us to approximately place the inducer relative to known species.
+
+## 2. Determine the gall's development timeline
+  Generally speaking, the most difficult part of collecting an inducer specimen is making your collection at the right time. To do that, you need to have a reasonable idea of when the inducer is likely to reach different points of its life cycle. If you find a gall that already has emergence holes, you're either too late or just in time. If there are no holes, it may be too early to collect.
+  There are a few ways to find out, and all of them are valuable even if you don't end up successfully rearing an inducer. 
+  If you can conveniently revisit the site, the most informative and least invasive is to simply check the gall at frequent intervals until you see evidence of emergence, which will give us at least one estimate of its emergence timing. 
+  If you aren't likely to see it again, you can either collect the gall and try to rear it (see below). If you succeed, great; if not, then we know to wait a bit longer next time. If you don't want to take it home, or you have a lot of galls in front of you, it's once again informative to cut one open to see what developmental stage the inducer is in. This also lets us better calibrate future collections.
+  
+## 3. Collect at the right time  
+  Once your gall is in the right stage for collection, depending on the taxon, you can either collect the sample directly or take the gall off the plant and bring it home to complete maturation. In every case, collecting more specimens is better for science and generally (but not necessarily or universally--use your discretion) not a threat to populations.
```

Comment:
For most galls you need to take part of the plant as well, i.e., small stem galls you need to cut the stem above the gall and take the whole stem, or many leaf galls you have to take the whole leaf.